packages/itwin/map-layers/src/ui/widget/TransparencyPopupButton.tsx

```diff
@@ -23,7 +22,7 @@ export interface TransparencyPopupButtonProps {
 }
 
 /** @alpha */
-// eslint-disable-next-line @typescript-eslint/naming-convention
+// eslint-disable-next-line @typescript-eslint/unbound-method
```

Comment:
what is unbound?

Reply:
the linter didn't like how `onTransparencyChange` wasn't an arrow function, it warned about unintentionally scoping `this` for that function. I removed this lint disable and changed the props to make that function be an arrow function instead